dev-requirements.txt

```diff
@@ -1,143 +1,50 @@
 -i https://pypi.python.org/simple
-apipkg==1.5
+apipkg==1.5; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
```

Comment:
In the past there has been a lot of resistance to having these version qualifiers in here.  I personally am not objecting it but maybe @KPrasch and @derekpierre want to weigh in.

Reply:
Yeah, those were automatically added by pipenv. I don't know if there's a flag we can use for this...

Reply:
I find it to be strange that this is the default behavior for the latest pipenv, but who am I to blow against the wind - there must be a good reason, right? :laughing:

Reply:
I have spent some time trying a lot of flags and have been unable to eliminate them.